tests/baselines/reference/jsDeclarationsInterfaces.js

```diff
@@ -164,6 +164,7 @@ export interface M<T_1> {
 }
 export interface N<U_1> extends M<U_1> {
     other: U_1;
+    field: U_1;

```

Comment:
this is incorrect, and preventing it might be the intent of the old rule -- but interface-extends-interface is explicitly illegal in Javascript, so I think it's fine to emit harmlessly incorrect .d.ts from it.

Reply:
It was the intent, yeah. I was testing the js declaration emitter on all TS constructs just to make sure the node builder worked on TS type structures, since, while _we_ never invoke it on them (since we just transform the original ts ast), the API _is_ public, so it _might_ get invoked on them by a 3rd party.